### ISSUE TITLE
"delete %path%" effect doesn't work sometimes

### ISSUE THREAD

**Issue by Lemrigue** [Olyno/skent#22]
## Describe your problem

"delete %path%" effect doesn't work sometimes, in the code below, sometimes it's written that the file has been deleted, but the file is still there, when I do it again it works, and sometimes it doesn't

## Versions

**Skript:** 2.5.3
**Server:** paper-571 1.16.5
**Skent:** 3.0 (but I think I had the problem with Skent 2.0 as well)

## Code

```	
loop all directories from dir path "backups":
	delete {_purge::*}
	set {_purge::*} to all files from loop-value
	if size of {_purge::*} > {@files_limits}:
		set {_purge::*} to sorted {_purge::*}
		loop (size of {_purge::*} - 3) times:
			{_purge::%loop-number%} is set
			send "&7backup &c%{_purge::%loop-number%}% &7deleted."
			delete file path "%{_purge::%loop-number%}%" #doesn't work sometimes
			wait 1 second
```

## Error

no error

**Comment by Olyno** [comment]
Hi, thanks you for the report. This issue is due to the async stuff which has not been solved in the 3.0 version. I'm sorry about t hat. Gonna be fix when possible!

**Comment by Olyno** [comment]
Closing this issue in favor of #34
